.travis.yml

```diff
@@ -2,25 +2,36 @@ langauge: ruby
 sudo: false
 cache: bundler
 
-branches:
-  only: master
-
-rvm:
-  - 2.3.7
-  - 2.5.3
-
-gemfile:
-  - gemfiles/rails_3.gemfile
-  - gemfiles/rails_4.gemfile
-  - gemfiles/rails_5.gemfile
+before_install:
+  - gem uninstall bundler -x || gem uninstall bundler -a || true
+  - gem install -v 1.16.2 bundler --no-rdoc --no-ri
 
 matrix:
-  exclude:
-  - rvm: 2.5.3
+  include:
+  - rvm: 2.3.7
     gemfile: gemfiles/rails_3.gemfile
+    env: TEST_APP_DIR=spec/app_rails3
+  - rvm: 2.3.7
+    gemfile: gemfiles/rails_4.gemfile
+    env: TEST_APP_DIR=spec/app_rails4
+  - rvm: 2.3.7
+    gemfile: gemfiles/rails_5.gemfile
+    env: TEST_APP_DIR=spec/app_rails5
+  - rvm: 2.5.5
```

Comment:
@zdennis should we run rails 3 and 2.5.5?

Reply:
or does that not work unless you're on rails LTS?

Reply:
It doesn’t work unless on LTS. :(

Reply:
ok cool